brand_stock_account/models/account_move.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2022 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo import api, models
+
+
+class AccountMove(models.Model):
+    _inherit = "account.move"
+
+    @api.model
+    def _fill_brand_id(self, values):
+        values = values.copy()
```

Comment:
why copy values?

Reply:
It's a good practice, never update dict given. In some case you could have issues.

Reply:
I don't see the benefit here, can you, please, point a link that explains it, for my general knowledge

Reply:
I don't have any link, it's not documented. But in any case, this is not a blocking point about PR approval.

Reply:
Yes indeed